tools/migrate-db/README.md

```diff
@@ -0,0 +1,178 @@
+# migrate-db
+
+A tool to migrate Celestia App databases from LevelDB to PebbleDB.
+
+## Overview
+
+This tool migrates all celestia-app databases from LevelDB format to PebbleDB format. PebbleDB offers better performance and scalability compared to LevelDB.
+
+## Installation
+
+```bash
+cd tools/migrate-db
+go build -o migrate-db
+```
+
+## Usage
+
+### Basic Migration
+
+```bash
+./migrate-db
+```
+
+This will:
+1. Create a backup of the entire `data` directory to `data_backup`
+2. Create a `data_pebble` directory in `~/.celestia-app/`
+3. Migrate all databases to PebbleDB format
+4. Verify the integrity of migrated databases
+5. Display instructions for completing the migration
+
+### Options
+
+- `--home <path>` - Specify custom home directory (default: `~/.celestia-app`)
+- `--dry-run` - Test the migration without making changes
+- `--no-backup` - Skip creating backup of data directory (not recommended)
+- `--cleanup` - Remove old LevelDB files after migration (use with caution)
+
+### Examples
+
+**Dry-run (test without changes):**
+```bash
+./migrate-db --dry-run
+```
+
+**Custom home directory:**
+```bash
+./migrate-db --home /custom/path/.celestia-app
+```
+
+## Migrated Databases
+
+The tool migrates the following databases:
+
+- `application.db` - Application state (usually the largest)
```

Comment:
do we also want to modify the application state to pebble? have we compared this exclusively? I know pebble is a lot faster for the block store but do we need to also check for the appliction?